Jenkinsfile

```diff
@@ -184,7 +188,7 @@ pipeline {
           steps {
             node('built-in') {
               script{
-            sh 'mvn test-compile failsafe:integration-test -DskipUnitTests=true'
+            sh 'mvn test-compile failsafe:integration-test -DskipUnitTests=true -DintTestProxyHost=jenkins-master-priv -DintTestProxyPort=8090 -DintTestHost=api.cat-test.iudx.io -DintTestPort=80'
```

Comment:
We don't need to perform the zap testing again here, so you can remove the proxy related configurations. i.e proxy host and proxy port.